devicon.json

```diff
@@ -1095,6 +1095,33 @@
         "color": "#F38020",
         "aliases": []
     },
+    {
+        "name": "cloudflareworkers",
+        "altnames": [
+            "Cloudflare Workers"
+        ],
+        "tags": [
+            "platform",
+            "serverless",
+            "Deploy",
```

Comment:
I believe we need to keep tags lower-case.
```suggestion
            "deploy",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient